content/markdown/01-basic-r/01-01-coding-basics.rmd

```diff
@@ -0,0 +1,118 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Coding Basics
+**Author:** Cole Brookson
+**Date:** 13 June 2022
+
+Fundamentally, our goal in using R is to ask our computer to help us answer the biological domain questions we have. In order for R to be useful to us in this sense, we have to know *how* to ask our computer to do these things. It is easiest to think of R as simply a interface between us and our computers. However, we must remember that *computers are not intelligent, they will only do EXACTLY what we tell them to do*. Therefore, we need to be specific about how we ask our computer to do things in service of answering our biological questions. This section covers some important concepts that will govern how we ask R to do things for us. 
```

Comment:
I might phrase this very first sentence differently. Maybe with a welcome first? What about:

"Fundamentally, throughout your courses and research in ecology and evolutionary biology, you will be using R to ask your computer to help you understand your data. This might be doing simple calculations, such as finding means and medians of organismal size, or it might be more sophisticated, such as looking for significant differences in growth rates or building models to predict biological responses. In order for R to be useful..."